recipes/oneloop/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set name = "oneloop" %}
+{% set version = "3.7.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/hameren/{{ name }}/get/v{{ version }}.tar.gz
+  sha256: e03ed0e852f360fb19803dc6aaf83812604059ff9ae6baac5d856dbdf9b8797d
+
+build:
+  skip: true  # [win]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('oneloop', max_pin='x.x') }}
+  script:
+    - sed -i "s|FC = gfortran|FC = $FC|g" Config
+    - sed -i "s|FFLAGS = -O|FFLAGS = $FFLAGS|g" Config
+    - python create.py dynamic
+    - mkdir -p $PREFIX/include/oneloop
+    - mv *.mod $PREFIX/include/oneloop/
+    - mv libavh_olo.so $PREFIX/lib/
+    - ./clean.sh
```

Comment:
I am not the maintainer of this code, I am just trying to package it for use, so I appreciate that this seems like a bizarre way to generate a Fortran shared library.

Reply:
Well... The "standard way" would be to hide all that in a makefile ;-p